lib/libutee/assert.c

```diff
@@ -41,7 +41,7 @@ void _assert_log(const char *expr __maybe_unused,
 				expr, file, line, func);
 }
 
-void __noreturn _assert_break(void)
+void _assert_break(void)
```

Comment:
What's the difference between having the attribute at the front compared to at the end?

With `__noreturn` on `_asseet_break()` depending on `NDEBUG` the compiler can determine that a certain function that uses `assert()` can't return.

This is a debug function so it doesn't matter if the compiler fails to optimize something.